src/types.rs

```diff
@@ -378,7 +378,7 @@ impl Value {
         }
     }
 
-    fn resolve_enum(self, symbols: &Vec<String>) -> Result<Self, Error> {
+    pub fn resolve_enum(self, symbols: &Vec<String>) -> Result<Self, Error> {
```

Comment:
Why does this one need to be public? Isn't it still only used in `Value::resolve`? :)

Reply:
Oh yes, it was a leftover from when I tried to special case enums instead of calling resolve. Good catch!